src/main/java/mekanism/common/integration/MekanismHooks.java

```diff
@@ -44,12 +42,15 @@ public final class MekanismHooks {
     public void hookConstructor() {
         ModList modList = ModList.get();
         CraftTweakerLoaded = modList.isLoaded(CRAFTTWEAKER_MOD_ID);
+        CuriosLoaded = modList.isLoaded(CURIOS_MODID);
+        if (CuriosLoaded) {
+            CuriosIntegration.addListeners(FMLJavaModLoadingContext.get().getModEventBus());
```

Comment:
If you are adding events like this might as well just modify `hookConstructor` to accept the mod event bus as a parameter.